scripts/cluster/create_multinode_cluster.sh

```diff
@@ -48,4 +50,4 @@ while true; do
     esac
 done
 
-$DIR/setup_master_node.sh
\ No newline at end of file
+$DIR/setup_master_node.sh $STOCK_CONTAINERD
```

Comment:
Just to clarify. I need this to set up stock clusters.

Reply:
Okay, I did rebase, so that's why the commit tree is different.